projects/plugins/jetpack/extensions/blocks/contact-form/components/jetpack-form-settings-dropdown.js

```diff
@@ -0,0 +1,70 @@
+import { DropdownMenu, Flex, FlexItem, Icon, MenuGroup, MenuItem } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { close, external } from '@wordpress/icons';
+import { map } from 'lodash';
+import { PluginIcon } from '../../../shared/icons';
+
+const JetpackFormSettingsDropdownContent = ( { responsesPath, settings, onClose, ...props } ) => {
+	const [ currentSettingsView, setCurrentSettingsView ] = useState( null );
+
+	if ( ! currentSettingsView ) {
+		return (
+			<>
+				<span className="form-settings-dropdown__hint">
+					{ __( 'CONNECT FORM RESPONSES TO', 'jetpack' ) }
+				</span>
+				<MenuGroup>
+					{ map( settings, ( setting, index ) => (
+						<MenuItem
+							className="form-settings-dropdown__item"
+							key={ index }
+							onClick={ () => setCurrentSettingsView( setting ) }
+						>
+							{ setting.icon }
+							{ setting.title }
+						</MenuItem>
+					) ) }
+				</MenuGroup>
+				<MenuGroup>
+					<MenuItem
+						href={ responsesPath }
+						icon={ external }
+						target="_blank"
+						onClick={ () => onClose() }
+					>
+						{ __( 'Manage form responses', 'jetpack' ) }
+					</MenuItem>
+				</MenuGroup>
+			</>
+		);
+	}
+
+	return (
+		<div style={ { padding: '12px' } }>
+			<Flex style={ { marginBottom: '24px' } }>
+				<FlexItem>
+					<strong>{ currentSettingsView.title }</strong>
+				</FlexItem>
+				<Icon icon={ close } onClick={ () => setCurrentSettingsView( null ) } size={ 16 } />
```

Comment:
Nit: should we wrap this in a `<button>` for accessibility reasons?

Reply:
✅